### ISSUE TITLE
Best way to extend the model to a new speaker

### ISSUE THREAD

**Issue by vishalbhavani** [mindslab-ai/assem-vc#23]
Hi,
I have a 15 min recording of a new speaker. I'd like to train assem-vc to perform any-to-one voice conversion. Based on my previous experience, the best and fastest way to do it would be to create a single speaker dataset and further fine-tune both the pre-trained VC decoder and pre-trained HIFI-GAN vocoder. 

1) Am I correct or is there a better way to do it?
2) How would the training steps change to fine-tune the given pre-trained models?

**Comment by wookladin** [comment]
Hello!
Well, I've never done fine-tuning on single-speaker datasets before.
Hence I'm not sure actually fine-tuning the VC decoder will work well.
If fine-tuning does not work well, it would be good to use VCTK or LibriTTS and the single-speaker dataset together.
On the other hand, HiFi-GAN worked well even with finetuning on single-speaker datasets in my experience.

Please refer to my answer and try it. thank you.

**Comment by vishalbhavani** [comment]
Hi @wookladin ,
1) As expected the single speaker fine-tuning for VC decoder resulted in overfitting because of low data. The lowest val loss ~0.6 still seems pretty high. What was the best val loss for your multi-speaker experiments?
![image](https://user-images.githubusercontent.com/87795115/133655682-06b2af63-c5cc-46e0-819a-ed10ff07f4ae.png)
2) GTA finetuning HI-FI GAN with the above model gave surprising results. The loss did not improve with training time. Is it because the decoder itself was not good enough to create decent gta mels?
![image](https://user-images.githubusercontent.com/87795115/133656117-1bf720a7-064d-49de-a595-29e0a8a82b9e.png)

P.S: I'm trying multi-speaker training now. I'll keep you posted on the results.

**Comment by wookladin** [comment]
Hi.
1. I've uploaded the validation loss graph of the VC decoder at issue #17 
Loss converges to around 0.2. It seems like your VC decoder is overfitted.
A multi-speaker setting will help you to avoid overtiffing.
Thank you for sharing the results!

2. Unfortunately, I am not sure by looking at the loss graph.
Did you hear the logged audios at the validation step?
The perceptual quality at that time seems to be important in judgment.

Thank you!

**Comment by vishalbhavani** [comment]
Hi @wookladin ,
Multi-speaker training solved the overfitting problem as expected. Logged audios in vocoder training also sound good after that. Thanks

**Comment by kannadaraj** [comment]
@vishalbhavani Thanks for the confirmation that it worked in your case. Did you warm start pre-trained model with the new speaker or did you train from scratch?

**Comment by vishalbhavani** [comment]
I started with the pre-trained model
